Rex.xcodeproj/project.pbxproj

```diff
@@ -395,6 +381,26 @@
 			path = Helpers;
 			sourceTree = "<group>";
 		};
+		9AB3BCAD1D3D23E500063B96 /* WatchKit */ = {
+			isa = PBXGroup;
+			children = (
+				9AB3BCAF1D3D23E500063B96 /* WatchKitExtensions.gyb.py */,
+			);
+			path = WatchKit;
+			sourceTree = "<group>";
+		};
+		9AB3BCBA1D3D243C00063B96 /* Generated Sources */ = {
+			isa = PBXGroup;
+			children = (
+				9AB3BCB61D3D241A00063B96 /* AppKitExtensions.generated.swift */,
+				9AB3BCAE1D3D23E500063B96 /* WatchKitExtensions.generated.swift */,
+				9AB3BCB21D3D23FD00063B96 /* UIKitExtensions.iOS.generated.swift */,
+				9AB3BCB31D3D23FD00063B96 /* UIKitExtensions.tvOS.generated.swift */,
```

Comment:
The best practice for generated source code files is to not add them to the project—but to generate them into `CONFIGURATION_BUILD_DIR` and use them from there.